content/logs/s3.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archives on AWS S3
+kind: Documentation
+description: "Forward all your Datadog logs to S3 for long term storage."
+---
+
+<div class="alert alert-warning">
+This feature is currently in Limited Beta. Ask your Sales representative or Customer Success Manager to have it enabled.
+</div>
+
+## S3 Archives
+
+Configure your Datadog account to forward all the logs ingested to your own S3 bucket in order to keep in long-term storage all the logs that you used Datadog to aggregate together. This guide shows you how to set this up. 
+ 
+## Create and Configure an S3 Bucket
+
+1. Go into your [AWS Console][1] and [create an S3 bucket][2] to send your archives to. Be careful not to make your bucket publicly readable. 
+
+2. Modify your bucket to grant write-only access to the Datadog AWS user. Do this by editing your bucket’s **permissions**, and setting the **bucket policy** with the following content (replace "my-bucket-name" with the name of your bucket):
+
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "AllowDatadogArchivesUploader",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "464622532012"
+                },
+                "Action": [
+                    "s3:PutObject",
+                    "s3:PutObjectAcl",
+                    "s3:ListMultipartUploadParts",
+                    "s3:AbortMultipartUpload"
+                ],
+                "Resource": "arn:aws:s3:::{{MY-BUCKET-NAME}}/*"
```

Comment:
Variables to change on config should [respect the Style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#conventions) 
The correct resource value is then: `arn:aws:s3:::{{<MY_BUCKET_NAME>}}/*` or `arn:aws:s3:::<MY_BUCKET_NAME>/*` ?